src/bindings/python/docs/code_examples.md

```diff
@@ -61,18 +61,18 @@ But how to extend existing API? Let's navigate to `openvino/src/bindings/python/
 
```

Comment:
I would like to review these files. For example why here: `openvino/src/bindings/python/src/openvino` we don't have a link? :)

Reply:
Feel free to do so, let me offline on your review. You can follow https://github.com/openvinotoolkit/openvino/pull/12145 for additional context.